native/templates/openharmony/entry/src/main/ets/MainAbility/pages/index.ets

```diff
@@ -51,14 +51,9 @@ struct Index {
             engineUtils.writablePathInit(data);
           });
 
-          // HACK: delay to post massage to wait for worker launching.
-          setTimeout(() => {
-            log('post msg when XCLoad');
-            console.log('cocos xcomponent onLoad');
-            this.context = context;
-            this.cocosWorker.postMessage({type: "onXCLoad", data: "XComponent"});
-          }, 500)
-
+          log('post msg when XCLoad');
```

Comment:
What's the difference between `log` and `console.log`?

Reply:
![WXWorkCapture_16618394291495](https://user-images.githubusercontent.com/5205637/187361531-4454d9c4-d552-4088-ab5a-8042cbc1d4e2.png)

In order to facilitate debugging, additional information will be added to this function before